composer.json

```diff
@@ -79,5 +79,13 @@
         "mutate": "infection --threads=2 --skip-initial-tests --coverage=build/phpunit",
         "style": "@cs-fix",
         "test": "phpunit"
+    },
+    "support": {
+        "forum": "https://forum.codeigniter.com",
+        "chat": "https://github.com/codeigniter4/shield/discussions",
```

Comment:
GitHub Discussions do not seem to be *chat*. It is *forum* ?

Reply:
Actually I don't know how to add two forum.
It is not explained here:
https://getcomposer.org/doc/04-schema.md#support

Reply:
You can't. `support.forum` should be a string. If you set an array, it raises a validation error.

Reply:
So which one do you choose between `https://forum.codeigniter.com` and `https://github.com/codeigniter4/shield/discussions`?

Reply:
I'm in favor of Discussions. The forums are kind of a mess of support issues and anything we can do to split them out a bit more might be good.

Reply:
@kenjis answers more support forum threads than me though, so he may disagree 😉

Reply:
Maybe this conversation will be useful for you in making a decision. Some people still have problems with the forum.

![Screenshot_2022-09-14-15-23-22-467_com miui gallery](https://user-images.githubusercontent.com/9530214/190135972-f25254df-c26d-42d3-a8d9-f8d7008c3d9d.jpg)

Reply:
I'm fine with GitHub discussions.
To be honest, I like GitHub because it is easier to read the source code than the forum.

Reply:
Good feedback.
Done!

Reply:
Our forums are hard to use, on many fronts. It's pretty old software, and even though it has been kept up it has that "patched legacy churn" feel. The team keeps throwing around the idea of building a CI4 forum software to replace it, as a group project, but everyone is already very busy and the framework takes lots of time so I don't know if that will happen.